packages/udes-ui-validation/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "udes-ui-validation",
+  "version": "0.0.1",
+  "description": "Package to validate react component fields.",
+  "main": "webpack.config.js",
+  "scripts": {
+    "jest:lerna": "jest --coverage",
+    "test": "lerna run jest:lerna",
+    "lint": "eslint src -c .eslintrc.json --ext js,jsx,ts,tsx; exit 0",
+    "check": "lint-staged"
```

Comment:
There should be a build alias here

Reply:
still working on making it an actual buildable package, this was just so that the tests and code could be reviewed =)